app/bundles/CoreBundle/Assets/js/4.builder.js

```diff
@@ -351,6 +351,7 @@ Mautic.closeBuilder = function(model) {
             // Trigger slot:destroy event
             document.getElementById('builder-template-content').contentWindow.Mautic.destroySlots();
 
+            var xs = new XMLSerializer();
```

Comment:
`serializeToString` is supported by all major browsers.
Only `serializeToStream` is not
![image](https://user-images.githubusercontent.com/18265735/27339765-f016925e-55d8-11e7-92a7-37929545457b.png)
![image](https://user-images.githubusercontent.com/18265735/27339817-1afb8f42-55d9-11e7-81e0-e28fc1da6b3a.png)
https://caniuse.com/#feat=xml-serializer